### DISCUSSION TITLE
Is there an SWC equivalent to `defaultLoaders.babel` passed to custom webpack config?

### DISCUSSION THREAD

**Discussion by stefanprobst** [vercel/next.js#30861] [Help] [answered]
Is there an SWC equivalent to `defaultLoaders.babel` which is passed to a custom webpack config in `next.config.mjs`?

**Comment by timneutkens** [comment, marked as answer]
When SWC is enabled `defaultLoaders.babel` refers to the SWC loader for backwards compat, so you can rely on that so it will be compatible between enabled/disabled

**Reply by stefanprobst** [reply]
cool thanks!
